app/helpers/application_helper/button/csv.rb

```diff
@@ -0,0 +1,5 @@
+class ApplicationHelper::Button::Csv < ApplicationHelper::Button::Basic
+  def check_rbac?
+    %w(render_report_csv chargeback_download_csv compare_download_csv).include?(self[:child_id]) ? role_allows?(:feature => self[:child_id]) : true
```

Comment:
My wrong. This is not the base class. So the list here is an exception. We should get rid of these exceptions where possible.